clang/test/OpenMP/error_unsupport_feature.c

```diff
@@ -0,0 +1,8 @@
+// RUN: %clang_cc1 -emit-obj -verify -fopenmp %s
```

Comment:
`emit-obj` is not needed

Reply:
during testing, I found it can't produce this error without `-emit-obj`

Reply:
You can use `-emit-llvm` instead.